src/registrar/templates/includes/footer.html

```diff
@@ -14,20 +14,23 @@
             <div class="mobile-lg:grid-col-auto">
               <img
                 class="usa-footer__logo-img"
-                src="{% static 'img/dottedgov-round.svg' %}"
+                src="{% static 'img/dotgov_logo.svg' %}"
```

Comment:
Image is not there in sandbox. 🙂 
![image](https://user-images.githubusercontent.com/15718530/235454425-143bd7e8-f0df-4e01-94aa-93953a1be976.png)

Reply:
Get me every time! Thanks for catching it.